internal/tui/list.go

```diff
@@ -14,6 +15,16 @@ import (
 	"github.com/shazow/wifitui/wifi"
 )
 
+const scanInterval = 10 * time.Second
```

Comment:
Let's move the scanInterval into the ListModel struct, and add a --scan-interval=10 cli flag that people can override.